AOJ/06/06_C.py

```diff
@@ -0,0 +1,13 @@
+n = int(input())                                                            
+list = [[[0 for i in range(10)] for j in range(3)] for k in range(4)]   #多次元配列の生成
```

Comment:
```suggestion
building = [[[0 for _ in range(10)] for _ in range(3)] for _ in range(4)]  # 多次元配列の生成
```
- listという変数名は、組み込み関数`list()` と名称が重複しているため、`building` 等のより意味のある変数名を使用しましょう。
- 内包型のリストで、カウンタ変数`i, j, k`は、呼び出されていないため、`_` など明示的に使用しないという意味のカウンタ変数に変えましょう。